src/main/java/org/gridsuite/network/map/model/TapChangerData.java

```diff
@@ -17,6 +19,44 @@
 @Getter
 @EqualsAndHashCode
 public class TapChangerData {
-    Integer lowTap;
-    Integer highTap;
+    private Integer lowTap;
+
+    private Integer tapPosition;
+
+    private Integer highTap;
+
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    private Boolean voltageRegulation;
+
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    private Boolean loadTapChangingCapabilities;
+
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    private Double targetVoltage;
```

Comment:
`targetV` maybe to keep coherent with other services ?

Reply:
✓